src/.env

```
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_APP_URL = 'https://sensors.africa/'
+NEXT_PUBLIC_GA_ID =  'G-8M50XTMKVG'
```

```suggestion
NEXT_PUBLIC_APP_URL = 'http://localhost:3000'
```
[`.env`](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#environment-variable-load-order) is for default values only i.e. when it doesn't matter if it's PROD or DEV or local machine.

- PROD domain should only be set in PROD (via dokku config:set)
- If we set GA_ID here, it means even in DEV (think Vercel) this ID will be used & hence will pollute the actual data. Therefore, it's best to also set it in PROD.

PS: Lets standardise how we name this variable `GA_ID` here, `GOOGLE_ANALYTICS_ID` in ui repo, etc., makes it harder to remember.